set.mm

```diff
@@ -354553,6 +354589,103 @@ be obtained in Metamath (without the need for a new definition) by the curry
       DWDOAWHDWEWFWG $.
   $}
 
+  $( The general logarithm of a positive real number with respect to a real
```

Comment:
The logarithm of a positive real number to a real base...

Reply:
Done